src/github.com/matrix-org/dendrite/roomserver/types/types.go

```diff
@@ -75,3 +83,15 @@ const (
 	// EmptyStateKeyNID is the numeric ID for the empty state key.
 	EmptyStateKeyNID = 1
 )
+
+// StateDataNIDList is used to return the result of bulk StateDataNID lookups from the database.
+type StateDataNIDList struct {
```

Comment:
I thought that the comment was sufficient here? (I'd like to keep "Result" available for when we start adding query APIs to the /api package that are used by other dendrite components to query room information from the room server)